### DISCUSSION TITLE
Can pytorch-accelerated be used together with slurm ？

### DISCUSSION THREAD

**Discussion by AlfredQin** [Chris-hughes10/pytorch-accelerated#47] [Q&A] [answered]
Hi, @Chris-hughes10 , thanks for your great work. I have been using  pytorch-accelerated for half year in my own machine and benefit a lot. I am wondering if you once test it in HPC with slurm,  and if so, could you please share an example of using slurm in this project ? Thank you !

**Comment by Chris-hughes10** [comment, marked as answer]
Hi @AlfredQin, I'm glad that you are PyTorch-accelerated useful. Honestly, I am not too familiar with slurm, as it isn't something that I use, so I may not be able to assist too much here. 

PyTorch-accelerated uses accelerate under the hood (which is just built on plain PyTorch), so any example that works there should also work here. I found [this thread](https://github.com/huggingface/accelerate/issues/145#issuecomment-1184817922 ) which includes an example, perhaps this would be useful as a starting point. 

If you are able to get it working with accelerate, or even plain PyTorch, I may be able to help further.
